string name
uint16 age
string color
